src/main/java/uk/gov/hmcts/reform/wacaseeventhandler/controllers/CaseEventHandlerLivenessHealthController.java

```diff
@@ -0,0 +1,73 @@
+package uk.gov.hmcts.reform.wacaseeventhandler.controllers;
+
+import lombok.extern.slf4j.Slf4j;
+import org.apache.commons.lang3.StringUtils;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.boot.actuate.availability.LivenessStateHealthIndicator;
+import org.springframework.boot.availability.ApplicationAvailability;
+import org.springframework.boot.availability.AvailabilityState;
+import org.springframework.boot.availability.LivenessState;
+import org.springframework.stereotype.Component;
+import uk.gov.hmcts.reform.wacaseeventhandler.entity.CaseEventMessageEntity;
+import uk.gov.hmcts.reform.wacaseeventhandler.services.CaseEventMessageCacheService;
+
+import java.time.LocalDateTime;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+@Slf4j
+@Component
+public class CaseEventHandlerLivenessHealthController extends LivenessStateHealthIndicator {
+
+    @Value("${management.endpoint.health.newMessageStateThresholdForLivenessCheck}")
+    private int newMessageStateThresholdForLivenessCheck;
+
+    @Value("${management.endpoint.health.newMessageLivenessStateCheckEnvEnabled}")
+    private String newMessageLivenessStateCheckEnvEnabled;
+
+    @Value("${environment}")
+    private String environment;
+
+    CaseEventMessageCacheService caseEventMessageCacheService;
+
+    @Autowired
+    public CaseEventHandlerLivenessHealthController(ApplicationAvailability availability,
+                                                    CaseEventMessageCacheService caseEventMessageCacheService) {
+        super(availability);
+        this.caseEventMessageCacheService = caseEventMessageCacheService;
+    }
+
+    @Override
+    protected AvailabilityState getState(ApplicationAvailability applicationAvailability) {
+        log.info("CaseEventHandler Liveness check Invoked for environment {} ", environment);
+        log.info("CaseEventHandler Liveness check configured for environments {} ",
+                 newMessageLivenessStateCheckEnvEnabled);
```

Comment:
Do we need these logs at info?  
I ask since there's a drive to reduce them and this is being probed every 15s or so.

Reply:
Yes, we can live with debug level log here. Made the change now.